subsys/bootloader/bl_validation/Kconfig

```diff
@@ -37,7 +37,7 @@ config SB_VALIDATION_INFO_CRYPTO_ID
 config SB_VALIDATION_INFO_VERSION
 	int "Version number of the fw_validation_info* structs (8 bits)."
 	range 0 255
-	default 2
+	default 3
```

Comment:
I'm not totally sure we need this. The incompatibility goes only one way: Old bootloaders do not support non-zero next_img_offset, but old bootloaders support the new format if next_img_offset is zero, and new bootloaders support the old format.

Maybe fw_info_check should be updated to support both 2 and 3?

Reply:
I don't see an issue here this bootloader is not upgradeable, so IMO we should fail "as often as we can" since its only a matter of re-compiling/re-flashing the new version AFAICS.

Reply:
The problem is if there are already products with an old bootloader. Those will not recognize new fw_infos if the version is incremented, even if those fw_infos are actually compatible (nex_img_offset is 0).